### PR TITLE
minimessage-text: added docs for line break (<br>)

### PR DESCRIPTION
**Pull request by JOO200**
This is the docs PR for https://github.com/KyoriPowered/adventure/pull/687

### CODE REVIEW

File: source/minimessage/format.rst [hunk 1]
```diff
@@ -270,3 +270,19 @@ Examples
...
 .. image:: /minimessage/images/font_1.png
    :alt: The result of parsing ``Nothing <font:uniform>Uniform <font:alt>Alt  </font> Uniform``, shown in-game in the Minecraft client's chat window
+
+New Line
+********
+
+Allows you to insert a new line
+
+Tag
+   :mm:`<font:key>`
```
Comment (KingOfSquares) [inline source/minimessage/format.rst:280]: Wrong tag

**Review by KingOfSquares — CHANGES_REQUESTED** [review]

**Comment by KingOfSquares** [comment]
Looking at

File: source/minimessage/format.rst [hunk 2]
```diff
@@ -270,3 +270,21 @@ Examples
 
 .. image:: /minimessage/images/font_1.png
    :alt: The result of parsing ``Nothing <font:uniform>Uniform <font:alt>Alt  </font> Uniform``, shown in-game in the Minecraft client's chat window
+
+New Line
```
Comment (kezz) [inline source/minimessage/format.rst:274]:
```suggestion
Newline
```

File: source/minimessage/format.rst [hunk 3]
```diff
@@ -270,3 +270,21 @@ Examples
...
+********
+
+Allows you to insert a new line
```
Comment (kezz) [inline source/minimessage/format.rst:277]:
```suggestion
Insert a newline character.
```

File: source/minimessage/format.rst [hunk 4]
```diff
@@ -270,3 +270,21 @@ Examples
...
+
+Tag
+   :mm:`<br>`
```
Comment (kezz) [inline source/minimessage/format.rst:280]: I'd swap this around, make `<newline>` be the default with `<br>` being the alias.

**Review by kezz — CHANGES_REQUESTED** [review]

**Review by kezz — APPROVED** [review]

**Pull request merged**